code/datums/traits/negative.dm

```diff
@@ -133,6 +133,8 @@
 				heirloom_type = /obj/item/reagent_containers/food/drinks/bottle/whiskey
 			if("Lawyer")
 				heirloom_type = pick(/obj/item/gavelhammer, /obj/item/book/manual/wiki/security_space_law)
+			if("Prisoner")
+				heirloom_type = /obj/item/pen/blue
```

Comment:
why

Reply:
At this point it's a placeholder. Any suggestion for a more appropriate alternative is welcome.

Reply:
instant noodles or a carrot shiv would work

Reply:
soap